WebApiCustomers/Repository/BaseRepository.cs

```diff
@@ -0,0 +1,73 @@
+﻿using FluentValidation.Results;
+using WebApiCustomers.Model;
+using WebApiCustomers.Validator;
+
+namespace WebApiCustomers.Repository
```

Comment:
```suggestion
namespace WebApiCustomers.Services
```
Alterar nome da pasta para Services